src/vs/workbench/api/browser/mainThreadTerminalService.ts

```diff
@@ -468,3 +458,10 @@ export function getOutputMatchForLines(lines: string[], outputMatcher: ITerminal
 	const match: RegExpMatchArray | null | undefined = lines.join('\n').match(outputMatcher.lineMatcher);
 	return match ? { regexMatch: match, outputLines: outputMatcher.multipleMatches ? lines : undefined } : undefined;
 }
+
+function parseQuickFix(id: string, source: string, fix: TerminalQuickFix): ITerminalQuickFix {
```

Comment:
when I try to test this, the window won't open with this error:

```
0x107c95ce0). One of the two will be used. Which one is undefined.
[25077:1201/113233.427542:INFO:CONSOLE(261)] "[uncaught exception]: Error: Cannot find module 'vscode'
Require stack:
- electron/js2c/renderer_init", source: vscode-file://vscode-app/Users/meganrogge/Repos/vscode/out/bootstrap-window.js (261)
[25077:1201/113233.428003:INFO:CONSOLE(264)] "Error: Cannot find module 'vscode'
Require stack:
- electron/js2c/renderer_init
    at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at i._resolveFilename (node:electron/js2c/renderer_init:33:1095)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at i._load (node:electron/js2c/renderer_init:33:356)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at nodeRequire (v
```

when I remove lines 463-464, it works

Reply:
a way around it is to instead check if `uri` in fix but not sure why the above wouldn't work

Reply:
Are you importing uri from `vscode`? You need to use the core lib